docs/introduction/example_scenarios.md

```diff
@@ -0,0 +1,72 @@
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import ThemedImage from '@theme/ThemedImage';
+
+# Example Scenarios
+
+Below, we present example scenarios that can be achieved with Jellyfish.
+Each scenario consists of a short description and illustrative Jellyfish 
+internal architecture so that you can catch the difference between peers
+and components even better!
+
+## Real-time video conferencing system
+
+In this scenario, we want to create a simple video conferencing system
+where people can talk to each other in the real-time.
+
+Beacuse of interactive and real-time requiremnts, the natural choice is
+to use WebRTC standard.
+
+The final architecture would look like this:
+
+<ThemedImage
+  alt="Docusaurus themed image"
```

Comment:
Nitpick, but we should have meaningful alt text(s)
```suggestion
  alt=" Jellyfish - Video conferencing architecture example"
```
Please go through other images as well